gr-uhd/grc/rfnoc.domain.yml

```diff
@@ -7,4 +7,10 @@ multiple_connections_per_output: false
 
 templates:
 -   type: [rfnoc, rfnoc]
-    connect: self.rfnoc_graph.connect(self.${ source.parent_block.name }.get_unique_id(), ${ source.key }, self.${ sink.parent_block.name }.get_unique_id(), ${ sink.key }, False)
+    connect: self.rfnoc_graph.connect(self.${ source.parent_block.name }.get_unique_id(), ${ source.key }, self.${ sink.parent_block.name }.get_unique_id(), ${ sink.key }, ${ is_back_edge })
+    parameters:
```

Comment:
Why does `parameters` come out indented under `templates`?

Reply:
The parameters are specific to this connection type (`[rfnoc, rfnoc]`).